---
course_id: 7-343-an-rna-safari-exploring-the-surprising-diversity-of-mammalian-transcriptomes-spring-2016
layout: course_section
menu:
  leftnav:
    identifier: 0eed205b0cf68252cb9c14da3e727f87
    name: Lecture Summaries
    weight: 20
title: Lecture Summaries
type: course
uid: 0eed205b0cf68252cb9c14da3e727f87

---

| WEEK # | TOPICS | LECTURE SUMMARIES |
| --- | --- | --- |
| 1 | Introduction and Course Overview |  {{< br >}}{{< br >}} {{< anchor "Week_1" >}}{{< /anchor >}}The semester will begin with general self-introductions by the instructors and student participants. The instructors will then: {{< br >}}{{< br >}} 1.  Present a general overview of the course format, the course website, and assignments,{{< br >}}2.  define objectives of the course, and{{< br >}}3.  describe what is expected from the student participants. {{< br >}}{{< br >}} We will start by giving a biological context for the RNA species that we will be working through in the course—introducing major concepts, techniques, and historical background that will be necessary for the first few sessions. Finally, we will discuss how to critically read scientific papers and access additional resources of the scientific literature. {{< br >}}{{< br >}}  |
| 2 | In the Beginning: Ribosomes and Ribosomal RNA |  {{< br >}}{{< br >}} {{< anchor "Week_2" >}}{{< /anchor >}}In the second week, we will begin our discussion in the 1950s. At that time, although the most prevalent biological macromolecules in cells (DNA, RNA, proteins, etc.) were known to exist, how they came about was unknown. Enter George Palade, an early cell biologist at the forefront of electron microscopy with a knack for continuously improving its techniques and producing ever-clearer images. Palade had already made careful studies of large organelles like the endoplasmic reticulum and mitochondria but now focused on another structure that was "particulate in nature and small in size." These small structures were often scattered throughout the cytoplasm. Intriguingly, many of these cells also had the curious ability to strongly attract and retain in their cytoplasm a basic dye suspected thought to be a marker of high RNA content. A few years later, Palade explicitly linked these two observations with biochemical experiments showing that "microsomes" (now called ribosomes) contained very large amounts of RNA. As microsomes had been earlier linked to cellular protein synthesis, this strongly implicated RNA as a player in protein production. {{< br >}}{{< br >}}  |
| 3 | Careful Measuring of Unhealthy Amounts of Radioactivity: A Role for tRNAs in Polypeptide Synthesis |  {{< br >}}{{< br >}} {{< anchor "Week_3" >}}{{< /anchor >}}The role for RNA in protein synthesis of course did not stop with ribosomal RNA. Using radioactively-labeled leucine, Hoagland and colleagues found that although labeled leucine ended up in proteins, it can also become covalently attached to an RNA species. This covalent attachment was dependent upon ATP, and the RNA species interacts with microsomes (ribosomes), providing strong evidence that the RNA was a carrier of amino acids during protein synthesis. Recent reports, like that from Presnyak and colleagues, demonstrate that tRNAs are more than just simple deliverers of amino acids. The availability of tRNAs in cells can influence transcript stability on a global scale. {{< br >}}{{< br >}}  |
| 4 | mRNA as an Information Shuttle |  {{< br >}}{{< br >}} {{< anchor "Week_4" >}}{{< /anchor >}}By 1958, we know that DNA in the nucleus contains the information to make proteins and that these proteins are synthesized by structures located in the cytoplasm. What is the means by which this information is relayed from the nucleus to the cytoplasm? In other words, what is the messenger? What is the nature of ribosomes? Is each ribosome specialized for the translation of one particular protein or are all sites of general translation? In the first paper, Astrachan and Volkin find that upon infection with bacteriophage, a new population of RNA rapidly appears in _E. coli_ cells. This population has a base composition different from that of RNA from uninfected cells. In the second paper, Brenner, Jacob, and Meselson use a bacteriophage system to show that an intermediate molecule that carries information from DNA to ribosomes is RNA-based and associates with ribosomes that existed before the infection occurred. This changed the view of ribosomes from potentially specialized complexes (i.e. one protein species per ribosome) to versatile molecular readers. {{< br >}}{{< br >}}  |
| 5 | The Genomics Revolution: An Introduction to High-throughput Sequencing |  {{< br >}}{{< br >}} {{< anchor "Week_5" >}}{{< /anchor >}}The papers we have discussed thus far have studied particular systems in great detail. The methods they used can inspire awe in the minds of modern molecular biologists in that simple experiments led to profound insights into fundamental biological principles and made predictions, many of which turned out to be right. Recently, many problems have been approached from the opposite direction: instead of studying one system or one RNA in great detail, study the regulation and activity of many RNA molecules at once to learn about the general principles that govern them. One approach, called high-throughput sequencing, has been very successful and has revealed many interesting properties of gene expression. In this class we will introduce the fundamental principles and methods behind high-throughput sequencing, and particularly focus on high-throughput methods to characterize the abundance and confirmations of RNA species (RNA-seq). Many of the papers we will discuss throughout the remainder of the course will rely on these methods. RNA-seq technology represented an advance towards more precise measurements and comparisons of gene expression, as highlighted in Mortazavi _et al._, as well as allowed for unbiased ascertainment of mRNA isoform conformations to assess mRNA splicing. In the second paper, we will discuss how RNA-seq can be used as a tool to explore other aspects of RNA biology beyond simple transcript abundance. Wan _et al_. use a combination of ribonucleases and sequencing to interrogate RNA secondary structure transcriptome-wide. {{< br >}}{{< br >}}  |
| 6 | Let's Mix it up: SnRNAs and Pre-mRNA Splicing |  {{< br >}}{{< br >}} {{< anchor "Week_6" >}}{{< /anchor >}}Although mRNA molecules shuttle information from the nucleus to sites of translation, there are many steps in between the birth of an mRNA molecule and its delivery of information to ribosomes. One of these steps is the removal of introns from the mRNA. This process is controlled in large part by the action of small nuclear RNAs (snRNAs). These RNA molecules are often a few hundred nucleotides long and complex with particular proteins to form small nuclear ribonucleoproteins (snRNPs). It is these complexes that form the backbone of the spliceosome, a large, multi-subunit complex that removes intronic sequences from pre-mRNA molecules. When snRNAs were first characterized by Lerner _et al_, their function was unknown; however, even then, the authors speculated that they might be involved in pre-mRNA splicing. More recent studies, e.g. Jia _et al_. have shown that mutations in snRNAs can drive aberrant splicing across the transcriptome with potentially severe consequences. {{< br >}}{{< br >}}  |
| 7 | Let it Sno: Ribosomal RNA Modification Directed by SnoRNAs |  {{< br >}}{{< br >}} {{< anchor "Week_7" >}}{{< /anchor >}}Although small nuclear RNAs and small nucleolar RNAs (snoRNAs) had been known to exist for years, in the mid 1990s their specific functions remained largely unknown. However, there were preliminary indications. Some, like the snoRNA U3, had been shown to be required for the endonucleolytic cleavages necessary for rRNA maturation. Others had been shown to be involved in methylation of specific bases in rRNA. This week, we will discuss the role of snoRNAs in the deposition of another RNA mark: Pseudouridinylation. In the first paper, Ganot _et al_. find that pseudouridinylation of specific sites in rRNA molecules is dependent on complementarity between rRNA and snoRNA sequences. In the second paper, Jack _et al_. show that impairment of this pseudouridinylation system results in ribosomes with reduced activities, an effect that manifests in specific human diseases. {{< br >}}{{< br >}}  |
| 8 | Small is Beautiful: MicroRNAs as Ubiquitous Regulatory Molecules |  {{< br >}}{{< br >}} {{< anchor "Week_8" >}}{{< /anchor >}}Until now, we have largely viewed RNA molecules as positive regulators of gene expression. That is, most of them are components of efficient and stable protein production machinery. However, just as RNA giveth, it can also taketh away. Starting from classical genetic studies, Ambros and colleagues discovered that a very small RNA transcribed from the _lin-4_ locus of _C_. _elegans_ is complementary to a regulatory region of a target gene, _lin-14_, and acts to repress translation of the _lin-14_ mRNA. This discovery revealed a new class of RNA molecules, called microRNAs, that are antisense to given seed regions in mRNA molecules (particularly in 3' untranslated regions) and can act as negative regulators of either mRNA levels or of mRNA translation into protein. This conserved mechanism provides a way for cells to efficiently fine-tune the levels of mRNA or protein in the cell. As described in the second paper, insight into the processing of mature microRNA molecules from longer primary microRNA transcripts has led to the discovery of regulatory peptides produced from microRNA precursors that enhance microRNA levels and function. {{< br >}}{{< br >}}  |
| 9 | Running Interference: SiRNA as an Experimental Tool |  {{< br >}}{{< br >}} {{< anchor "Week_9" >}}{{< /anchor >}}RNA molecules that are antisense to mRNA molecules can act to antagonize the function of their cognate mRNAs. In a classic experiment, Fire and Mello use different combinations of sense / antisense RNAs with complementarity to various regions of specific transcripts and discover that delivery of a double-stranded RNA molecule is a potent repressor of expression. Specifically, injection of either sense or antisense RNA into _C._ _elegans_ resulted in only modest interference of endogenous genes; however, injection of sense and antisense RNA together resulted in potent interference. These molecules must have complementarity to _exonic_ portions of the gene for maximal effect. This discovery of RNAi (RNA interference) formed the basis for both the use of siRNAs (short interfering RNAs) as experimental tools to specifically (more or less) silence certain genes as well as the possibility of RNA-based treatments for human disease. {{< br >}}{{< br >}} [Field Trip]({{< baseurl >}}/sections/syllabus#Field_Trip) to Broad Genetic Perturbation Platform (as described in Syllabus). {{< br >}}{{< br >}}  |
| 10 | Protecting us from Ourselves: PiRNAs Silence Transposons in Germline Tissue |  {{< br >}}{{< br >}} {{< anchor "Week_10" >}}{{< /anchor >}}Not only are RNA molecules responsible for the expression and regulation of information stored in DNA, they also play in a role in protecting the integrity of the DNA. Transposable elements are constantly hopping from site to site within the genome, potentially causing havoc with each insertion. This phenomenon can be particularly deleterious in the germline, where such insertions can be passed to progeny with potentially disastrous effects. To the rescue come another class of small RNAs: the piRNAs. These RNAs are able to specifically recognize RNA produced from transposable elements and prevent their expression through destruction of the message. Not only that, some piRNAs have the ability to then enter the nucleus and direct heterochromatin formation over transposon loci, extending their protective abilities to future generations. Brennecke and colleagues show that _Drosophila_ piRNAs can silence transposons through a mechanism that rapidly amplifies piRNA production to efficiently inhibit transposon activity. Klattenhoff and colleagues then show that the Rhino protein is essential for the efficient production of these piRNAs and their transposon-silencing ability. {{< br >}}{{< br >}}  |
| 11 | Who Needs Proteins?: Long Noncoding RNAs as Regulators of Gene Expression |  {{< br >}}{{< br >}} {{< anchor "Week_11" >}}{{< /anchor >}}Although the most glamorous catalytic and structural molecules in the cell are often proteins, any RNA biologist will tell you that RNA molecules have been fulfilling these roles for billions of years. RNA molecules may be ideal scaffolds for regulatory complexes, particularly those that involve the regulation of gene expression, as they may contain both binding sites for many proteins and the ability to hybridize with other nucleic acids. We have seen that RNA molecules have the ability to silence expression at the level of RNA. As described in the first paper, the long noncoding RNA (lncRNA) Xist takes this repression to the level of DNA by repressing gene expression across almost an entire chromosome. By acting as a scaffold for transcriptional repressors and chromatin-modifying enzymes, Xist is able to efficiently shut off transcription of many genes at once. Further characterization of other lncRNAs has led to the possibility that they have many varied functions. As described in the second paper, the _Braveheart_ lncRNA has been observed to activate transcription networks by mediating the epigenetic regulation of cardiac development. {{< br >}}{{< br >}}  |
| 12 | No End in Sight: Circular RNAs and their (possible) Functions |  {{< br >}}{{< br >}} {{< anchor "Week_12" >}}{{< /anchor >}}The RNA molecules we have considered thus far have been linear RNAs, with possible structural complexities at the level of three-dimensional folding, or secondary structure. The pool of such RNA molecules is frequently turned over by regulated RNA degradation pathways targeting linear molecules. This paradigm has been thrown for a loop with the discovery of circular RNA species (circRNA), identified from high-throughput RNA sequencing data that showed a number of molecules with junctions between splice-sites that were not linearly sequential. In the first paper, Salzman and colleagues describe the identification and abundance of many circular RNAs in diverse cell types in humans though computational analyses of high-throughput sequencing data. In the second paper, detailed characterization of one such circRNA in humans, CDR1as, led to the idea that this long-lived RNA serves as a cellular sponge that is able to stably bind miRNAs and antagonize miRNA-based post-transcriptional regulation. Although later studies claimed to have identified dozens more such circRNA molecules, evidence for their functional relevance remains unclear. {{< br >}}{{< br >}}  |
| 13 | A New Hope: CRISPR-Associated RNAs, Prokaryotic Defense Mechanisms, have been Transformed into a Tool for Genome Editing |  {{< br >}}{{< br >}} {{< anchor "Week_13" >}}{{< /anchor >}}We have seen evidence that RNAs play large regulatory roles _in vivo_ and can be powerful experimental tools. The latest new RNA species to make waves are the CRISPR-associated RNAs (crRNAs), which have recently gained prominence as critical components of the CRISPR-Cas9 genome editing system. The biological basis for the CRISPR-Cas9 system is a naturally occurring prokaryotic adaptive immune process in which the Cas9 protein uses clustered regularly-interspaced short palindromic repeats (CRISPRs) derived from phage or other invading genomes to identify and cleave the corresponding invading DNA sequences. The adaptation of this system within eukaryotic cells and organisms has created a powerful and fast-growing molecular technique that has been used to edit genomes for experimental and potential therapeutic purposes. In the first paper, Cong and colleagues show that Cas9 can be programmed and targeted by crRNAs to direct genome editing in mammals. In the second paper, Hart and colleagues use this technique to target thousands of genes for loss-of-function analysis, finding over 1500 genes that are essential for fitness in human cells. {{< br >}}{{< br >}}  |
| 14 | Oral Presentations and Course Discussion | {{< anchor "Week_14" >}}{{< /anchor >}}In addition to student oral presentations this week, we will end class with a discussion of the course as a way for students to provide feedback. Course evaluations will be completed.